docs/source/ruletypes.rst

```diff
@@ -510,6 +522,74 @@ This value is added in back of the event. For example,
 
 ``kibana4_end_timedelta: minutes: 2``
 
+use_kibana_discover
```

Comment:
This is currently serving dual purpose of enabling and specifying the version.  It might be better to separate this into 2 configs:

``use_kibana_discover: true``
``kibana_discover_version: '7.3'``

Thoughts?

Reply:
Actually i think i want to take this path.